pythnet/remote-executor/cli/src/main.rs

```diff
@@ -0,0 +1,271 @@
+#![deny(warnings)]
+pub mod cli;
+
+use std::str::FromStr;
+
+use anchor_client::anchor_lang::{
+    AccountDeserialize,
+    AnchorDeserialize,
+    AnchorSerialize,
+    InstructionData,
+    Owner,
+    ToAccountMetas,
+};
+use clap::Parser;
+use cli::{
+    Action,
+    Cli,
+};
+
+use anyhow::Result;
+use remote_executor::{
+    accounts::ExecutePostedVaa,
+    EXECUTOR_KEY_SEED,
+    ID,
+};
+use solana_client::rpc_client::RpcClient;
+use solana_sdk::{
+    instruction::{
+        AccountMeta,
+        Instruction,
+    },
+    pubkey::Pubkey,
+    signature::{
+        read_keypair_file,
+        Keypair,
+    },
+    signer::Signer,
+    system_instruction,
+    system_instruction::transfer,
+    transaction::Transaction,
+};
+use wormhole_solana::{
+    instructions::{
+        post_message,
+        post_vaa,
+        verify_signatures_txs,
+        PostVAAData,
+    },
+    Account,
+    Config,
+    FeeCollector,
+    GuardianSet,
+    VAA as PostedVAA,
+};
+
+use remote_executor::state::{
+    governance_payload::{
+        ExecutorPayload,
+        GovernanceHeader,
+    },
+    posted_vaa::AnchorVaa,
+};
+use wormhole::VAA;
+
+fn main() -> Result<()> {
+    let cli = Cli::parse();
+
+    match cli.action {
+        Action::PostAndExecute { vaa, keypair } => {
+            let payer =
+                read_keypair_file(&*shellexpand::tilde(&keypair)).expect("Keypair not found");
+            let rpc_client =
+                RpcClient::new_with_commitment("https://pythnet.rpcpool.com/", cli.commitment);
+
+            let vaa_bytes: Vec<u8> = base64::decode(vaa)?;
+            let wormhole = AnchorVaa::owner();
+
+            let wormhole_config = Config::key(&wormhole, ());
+            let wormhole_config_data =
+                Config::try_from_slice(&rpc_client.get_account_data(&wormhole_config)?)?;
+
+            let guardian_set = GuardianSet::key(&wormhole, wormhole_config_data.guardian_set_index);
+            let guardian_set_data =
+                GuardianSet::try_from_slice(&rpc_client.get_account_data(&guardian_set)?)?;
+
+            let signature_set_keypair = Keypair::new();
+
+            let vaa = VAA::from_bytes(vaa_bytes.clone())?;
+
+            // RENT HACK STARTS HERE
+            let signature_set_size = 4 + 19 + 32 + 4;
+            let posted_vaa_size = 3 + 1 + 1 + 4 + 32 + 4 + 4 + 8 + 2 + 32 + 4 + vaa.payload.len();
```

Comment:
suggest documenting where these numbers came from and under what conditions we would have to change them.